src/section-solving-multistep-linear-equations-and-inequalities.mbx

```diff
@@ -513,20 +484,21 @@
 
 
     <subsection>
-        <title>Solving Multi-Step Inequalities</title>
-        <p>When solving a linear inequality, we follow the same steps in <xref ref="list-solve-linear-equation-parentheses">List</xref>. The only difference is that when we multiply or divide by a negative number on both sides of an inequality, the direction of the inequality symbol must switch. We will look at some examples.</p>
+        <title>Solving Multistep Inequalities</title>
+        <p>When solving a linear inequality, we follow the same steps in <xref ref="list-solve-multistep-linear-equation">List</xref>. The only difference in our steps to solving is that when we multiply or divide by a negative number on both sides of an inequality, the direction of the inequality symbol must switch. We will look at some examples.</p>
 
-        <list xml:id="list-solve-linear-inequality">
+        <list xml:id="list-solve-multistep-linear-inequality">
```

Comment:
In this list, you are using an `ol` with boldfaced words at the start of each item. I have yet to see a situation where actually using the `b` tag is the right move, and I'd recommend that if you are considering using it, that there is probably a better way. In this case, use a `dl` (description list) instead of an `ol`. Inside each `li`, give a `title`. Followed by one or more `p` to make the item content. In case the layout doesn't look good after doing this, try `width="narrow"`, `width="medium"`, or `width="wide"` as an attribute for the `dl`.

I'll let you try doing this instead of editing it in place.